vendor/github.com/emicklei/go-restful/examples/google_app_engine/datastore/main.go

```diff
@@ -125,8 +126,7 @@ func (u *ProfileApi) insert(r *restful.Request, w *restful.Response) {
 	}
 
 	// Let them know the location of the newly created resource.
-	// TODO: Use a safe Url path append function.
-	w.AddHeader("Location", u.Path+"/"+k.Encode())
+	w.AddHeader("Location", path.join(u.Path, k.Encode()))
```

Comment:
We cannot manually change vendored deps code. Only `dep` should manage them.

Reply:
ok for now i will revert this section of code to previous one